tooling/reproducible/ReproducibleBuilds.md

```diff
@@ -101,7 +148,13 @@ java -XshowSettings:
 ...
 ```
 
-eg.
+In cygwin, you must handle the trailing `\r` otherwise it will fail later:
+
+```bash
+   JAVA_VENDOR="$($JDK_DIR/bin/java -XshowSettings 2>&1| grep 'java.vendor = ' | sed 's/.* = //' | sed 's/\r.*//' )"
```

Comment:
The other URLs ought to be handled if we're doing this

Reply:
Thsi was just example.  In addition targeted to of removing the `\r` You have listing of all fo them in th e readem. I would not repeat them all here. I fyou insists, I will copypaste them all.

Reply:
I think just remove this, this is assuming someone is using a "script" to get these, when I wrote this I was simply running the command, and copy and pasting from the terminal, hence "\\r" does not come into it...
Lets keep this doc simple please

Reply:
Sure! As you wish. Removed. I had kept jsut inline "sed `\r`  away as eg: `sed 's/\r.*//'` is usually enough."